Service/ContentTypeService.php

```diff
@@ -409,4 +416,60 @@ public function getAllTypes()
         $this->loadEnvironment();
         return implode(',', array_keys($this->contentTypeArrayByName));
     }
+    
+    /**
+     * Export a content type in Json format
+     *
+     */
+    public function createContentTypeFromJson(File $jsonFile, Environment $environment)
+    {
+        $em = $this->doctrine->getManager();
+        /** @var ContentTypeRepository $contentTypeRepository */
+        $contentTypeRepository = $em->getRepository('EMSCoreBundle:ContentType');
+        $fileContent = file_get_contents($jsonFile->getRealPath());
+        
+        $encoders = array(new JsonEncoder());
+        $normalizers = array(new JsonNormalizer());
+        $serializer = new Serializer($normalizers, $encoders);
+        /**@var ContentType $contentType */
+        $contentType = $serializer->deserialize(
+            $fileContent,
+            "EMS\CoreBundle\Entity\ContentType",
+            'json'
+        );
+        $contentType->setEnvironment($environment);
+        $contentType->setActive(false);
+        $contentType->setDirty(true);
+        $contentType->getFieldType()->updateAncestorReferences($contentType, null);
+        $contentType->setOrderKey($contentTypeRepository->maxOrderKey() + 1);
+        
+        $this->persist($contentType);
+        return $contentType->getId();
```

Comment:
I think that your should return the $contentType object, if another function only needs the id, its that other function that should call the getId() function.